plugin/trino-openlineage/src/test/java/io/trino/plugin/openlineage/TestOpenLineageListener.java

```diff
@@ -61,6 +61,11 @@ void testGetCompleteEvent()
                 .extracting(RunEvent::getJob)
                 .extracting(Job::getNamespace)
                 .isEqualTo("trino://testhost");
+
+        assertThat(result)
+                .extracting(RunEvent::getJob)
+                .extracting(Job::getName)
+                .isEqualTo("queryId");
```

Comment:
Could explain why here equals to `queryId` instead of the value of the `queryId`?

Reply:
This is the value of `queryId` field in unit test:
https://github.com/trinodb/trino/blob/4ded3e20d160a2f021b70d2a50ab0d3da02ea315/plugin/trino-openlineage/src/test/java/io/trino/plugin/openlineage/TrinoEventData.java#L83